net.brinkervii.grapejuice.yml

```diff
@@ -0,0 +1,122 @@
+      # edit robloxplayer.desktop
+      - desktop-file-edit /app/share/applications/net.brinkervii.grapejuice.robloxplayer.desktop --set-icon="net.brinkervii.grapejuice.robloxplayer" --set-key="Exec" --set-value="grapejuice player %u"
+      # edit robloxstudio.desktop
+      - desktop-file-edit /app/share/applications/net.brinkervii.grapejuice.robloxstudio.desktop --set-icon="net.brinkervii.grapejuice.robloxstudio" --set-key="Exec" --set-value="grapejuice studio %u" 
+      # install grapejuice icons
+      - mkdir -p /app/share/icons/hicolor
+      - mkdir -p /app/share/icons/hicolor/64x64/apps/
+      - mkdir -p /app/share/icons/hicolor/128x128/apps/
+      - mkdir -p /app/share/icons/hicolor/scalable/apps/
+      # 64x64 icons
+      - cp src/grapejuice_common/assets/icons/hicolor/64x64/apps/grapejuice-roblox-player.png /app/share/icons/hicolor/64x64/apps/net.brinkervii.grapejuice.robloxplayer.png
+      - cp src/grapejuice_common/assets/icons/hicolor/64x64/apps/grapejuice-roblox-studio.png /app/share/icons/hicolor/64x64/apps/net.brinkervii.grapejuice.robloxstudio.png 
+      - cp src/grapejuice_common/assets/icons/hicolor/64x64/apps/grapejuice-roblox-player.png /app/share/icons/hicolor/64x64/apps/net.brinkervii.grapejuice.robloxapp.png
+      - cp src/grapejuice_common/assets/icons/hicolor/64x64/apps/grapejuice.png /app/share/icons/hicolor/64x64/apps/net.brinkervii.grapejuice.png
+      # 128x128 icons     
+      - cp src/grapejuice_common/assets/icons/hicolor/128x128/apps/grapejuice-roblox-player.png /app/share/icons/hicolor/128x128/apps/net.brinkervii.grapejuice.robloxplayer.png
+      - cp src/grapejuice_common/assets/icons/hicolor/128x128/apps/grapejuice-roblox-studio.png /app/share/icons/hicolor/128x128/apps/net.brinkervii.grapejuice.robloxstudio.png 
+      - cp src/grapejuice_common/assets/icons/hicolor/128x128/apps/grapejuice-roblox-player.png /app/share/icons/hicolor/128x128/apps/net.brinkervii.grapejuice.robloxapp.png
+      - cp src/grapejuice_common/assets/icons/hicolor/128x128/apps/grapejuice.png /app/share/icons/hicolor/128x128/apps/net.brinkervii.grapejuice.png
+      # scalable icons      
+      - cp src/grapejuice_common/assets/icons/hicolor/scalable/apps/grapejuice.svg /app/share/icons/hicolor/scalable/apps/net.brinkervii.grapejuice.svg
+      # install metainfo
+      - mkdir -p /app/share/metainfo/
+      - install -Dm644 net.brinkervii.grapejuice.metainfo.xml /app/share/metainfo 
+    sources:
+      - type: file
+        url: https://gitlab.com/brinkervii/wine-builds/-/raw/main/debuntu-wine-tkg-staging-fsync-git-7.1.r2.gc437a01e.tar.zst?inline=false
+        sha512: b60dd9628a592bef596049ff144ccdb7c2518afdbc70e32464f0d519122de4b2870ad8b7049f91acb876d302cde9189e899d54b615136d3bae1c0cc1bc2810da
+      - type: git
+        url: https://gitlab.com/brinkervii/grapejuice.git
+        tag: v5.1.1
+        commit: 6811a3776891a52e5569731ecf4d6937b5e4e755
+      - type: dir
```

Comment:
I personally prefer individual file entries. and patches over replacements.

Reply:
I can try to make patches where I can, when I get back in a couple of hours.

Reply:
done